### PR TITLE
Fixed a few console errors in JS tests

### PR DESCRIPTION
**Pull request by octaedro**
This PR fixes a few console errors that happen when running the JS tests.

Since we still have a couple of remaining console errors, I created [this issue](https://github.com/woocommerce/woocommerce-admin/issues/6473) to solve them in the next cooldown period. 

### Detailed test instructions:

- Check out the `main` branch.
- Verify the tests fixed here are showing console errors. You can run these tests individually by running:
`npm run test packages/components/src/plugins/test/index.js client/dashboard/components/connect/test/index.js client/homescreen/activity-panel/stock/test/index.js`

- Check out this branch (`git checkout fix/js_tests`).
- Verify the fixed tests now don't show console errors.

No changelog

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -84,6 +84,7 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 - Dev: Deprecate Onboarding::has_woocommerce_support. #6401
 - Fix: Broken link anchors to online documentation. #6455
 - Dev: Add Dependency Extraction Webpack Plugin #5762
+- Fix: Console errors in JS tests #6472
```
[the remark this answers is not in the document]
Comment (octaedro) [reply readme.txt]: Good idea! I removed it in the commit 45aa502

**Comment by octaedro** [comment]
Hi @samueljseay,

I just removed the changelog, could you give this PR another look?

**Review by louwie17 — APPROVED** [review]
LGTM 🚀 
@samueljseay feel free to give this a once over, but changelog entry has been removed.

**Review by samueljseay — APPROVED** [review]
🚀

**Pull request merged**